### PR TITLE
Add returned compiled_circuit to ibmq_compile`

### PR DESCRIPTION
**Pull request by vtomole**
Will update notebook once this is good to merge
Screenshot from 2022-03-06 16-07-10
.

### CODE REVIEW

**Review by vtomole — COMMENTED** [review]
Need to update `daily_integration` test.

**Comment by vtomole** [comment]
I've updated the notebook.

File: cirq_superstaq/service.py [hunk 1]
```diff
@@ -350,15 +350,19 @@ def ibmq_compile(
...
         try:
+            compiled_circuits = cirq_superstaq.serialization.deserialize_circuits(
+                json_dict["cirq_circuits"]
+            )
             pulses = applications_superstaq.converters.deserialize(json_dict["pulses"])
         except ModuleNotFoundError as e:
             raise applications_superstaq.SuperstaQModuleNotFoundException(
                 name=str(e.name), context="ibmq_compile"
             )
+        from cirq_superstaq import compiler_output
```
[the remark this answers is not in the document]
Comment (vtomole) [reply cirq_superstaq/service.py:361]: Done.

[the remark this answers is not in the document]
Comment (richrines1) [reply cirq_superstaq/service.py:353]: cool, can this import be removed now? (and same for other two instances in this file)

File: setup.py [hunk 2]
```diff
@@ -33,7 +33,7 @@
     author_email="pranav@super.tech",
     python_requires=(">=3.7.0"),
     install_requires=requirements,
-    license="N/A",
+    license="Apache 2",
```
Comment (richrines1) [inline setup.py:36]: did this PR absorb #191 ?

File: cirq_superstaq/service.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (vtomole) [reply cirq_superstaq/service.py]: Whoops. Done.

File: setup.py (hunk 2 above)
Comment (vtomole) [reply setup.py:36]: Yes. I accidentally made your requested changes to that PR. Thought merging it into here will make things easy.

**Review by richrines1 — APPROVED** [review]
lgtm!

**Pull request merged**